package-scripts.js

```diff
@@ -64,7 +64,7 @@ module.exports = {
     dev: {
       script: series([
         startInNewWindow('npm start dev.mongo --silent'),
-        startInNewWindow('cross-env PORT=8080 npm start dev.client --silent'),
+        startInNewWindow('./node_modules/.bin/cross-env PORT=8080 npm start dev.client --silent'),
```

Comment:
It looks like `npm start format` only touches "api/**/*.js" & "client/**/*.js". I ran prettier-eslint against `package-scripts.js` directly and while it goes to a new line, it still fails eslint's 80-char rule.